tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
-envlist = py27,py35,py36,pypy2
+envlist = py27,py35,py36,py37,pypy2
 
 [testenv]
-deps = pytest
-       httpretty
-commands = pytest -s tests/ integration_tests/
+extras = tests
+commands = pytest -sv tests/ integration_tests/
```

Comment:
I know this commit is just 4 lines, but this is 3 separate changes

* adding py3.7
* adding `-v`
* removing httppretty

all changes are trivial, but all (except py 3.7) should come with some explanation (eg i should know that httppretty was not used, or that -v is needed to see certain info in test output)

would you be able to split them into 3 commits, each with such explanation?

i know this is additional work for you, but this is just in case we need to return back to those commits looking for  "why so?"

Reply:
A more than fair request. :). I'll restage them tomorrow!